src/main/kotlin/com/example/fhirvalidator/controller/ValidateController.kt

```diff
@@ -16,19 +18,22 @@ class ValidateController(
         private val validator: FhirValidator,
         private val messageDefinitionApplier: MessageDefinitionApplier
 ) {
-    @PostMapping("/\$validate", produces = ["application/json"])
+    companion object : KLogging()
+
+    @PostMapping("/\$validate", produces = ["application/json", "application/fhir+json"])
```

Comment:
not following why `fhir+json` is needed - wont the jsonparser always encode the resource before returning it?